lib/app/utils/terminology.rb

```diff
@@ -220,13 +222,18 @@ def self.load_validators(directory = 'resources/terminology/validators/bloom')
           bfilter.include? probe
         end
         FHIR::DSTU2::StructureDefinition.validates_vs(validator[:url], &validate_fn)
+        @@loaded_validators[validator[:url]] = validator[:count]
       end
     end
 
     def self.get_valueset(url)
       @known_valuesets[url].valueset || raise(UnknownValueSetException, url)
     end
 
+    def self.loaded_validators
```

Comment:
could make this an `attr_reader` with an instance variable.

Reply:
See above.